apps/lobe-chat/1.41.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.41.0
+    image: lobehub/lobe-chat:v1.42.1
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference is the version of the Docker image used from `v1.41.0` to `v1.42.1`. This change could potentially include new features, security patches, bug fixes, or performance improvements. If you need more specific information about these updates, I'd recommend checking the [Lobebot documentation](https://docs.lobechat.com) for details on what changes have been made between those versions.